### PR TITLE
Adding homework #6

### PR DESCRIPTION
**Pull request by ValentynaVel**
I'm sending my homework #6. I have a question about lines 40-42: how to print key + value of 'contact_to_show'? I can print only phone number.

### CODE REVIEW

File: homework6.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+my_contacts = {
+    'Wednesday Addams': '+442255667788',
+    'Gregory House': '+112233445566',
+    'Winston Churchill': '+445566778899'
+}
+
```
Comment (klymenok) [inline homework6.py:14]: Add `while True` here to have an ability to run not only one command but use it as a running program

File: homework6.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# format "command amount"
+
+if command == 'stats':
+    print(len(my_contacts))
+    # calculating the number of contacts
+
+elif command == 'add':
+    new_contact_name = input('Type name and surname: ')
+    new_contact_phone = input('Type phone number: ')
+    my_contacts[new_contact_name] = new_contact_phone
```
Comment (klymenok) [inline homework6.py]: You should check if the key already exists. It is not allowed to override records

File: homework6.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    print(my_contacts)
+    # adding new contact
+
+elif command == 'delete':
+    contact_to_delete = input('Enter name and surname: ')
+    del my_contacts[contact_to_delete]
```
Comment (klymenok) [inline homework6.py]: It is better to check if key exists before trying to delete it

**Review by klymenok — CHANGES_REQUESTED** [review]

File: homework6.py (hunk 1 above)
Comment (ValentynaVel) [reply homework6.py:14]: As I understood correctly I have created infinite loop to make a running program

File: homework6.py (hunk 3 above)
Comment (ValentynaVel) [reply homework6.py]: I have also added check to "show" command

**Comment by ValentynaVel** [comment]
All requested changes were made.

File: homework6.py (hunk 1 above)
Comment (klymenok) [reply homework6.py:14]: That is correct

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**